ur_simulation_gazebo/launch/ur_sim_control.launch.py

```diff
@@ -127,6 +127,14 @@ def launch_setup(context, *args, **kwargs):
     )
     robot_description = {"robot_description": robot_description_content}
 
+    # define update rate
+    update_rate_config = PathJoinSubstitution(
+        [
+            FindPackageShare(runtime_config_package),
+            "config" + "update_rate.yaml",
+        ]
+    )
+
```

Comment:
This part should not be needed, because there is already `update_rate` parameter in the above yaml file.

Reply:
absolutely!